composer.json

```diff
@@ -52,6 +52,8 @@
         "matthiasnoback/symfony-dependency-injection-test": "^4.1",
         "phpstan/phpstan": "^0.12.29",
         "phpstan/phpstan-symfony": "^0.12.10",
+        "phpunit/phpunit": "^9.4",
+        "psalm/plugin-phpunit": "^0.13.0",
```

Comment:
Can you explain why you need to add this / update the psalm baseline ?

Reply:
I'll check that again (not really familiar with Psalm). Anyhow, I think we just fixed 2 of those issues by moving the exceptions inside the `getFilterService` and `getRendererService` functions.

Reply:
Yes, now I understand the psalm baseline change.

But now the plugin added since you're not even using it I think.